src/renderer/qssWidget/js/qssWidget.js

```diff
@@ -227,45 +251,39 @@
         });
     };
 
-    /**
-     * Determines the type of the `widget` subcomponent (either a menu or a stepper)
-     * depending on the type of the setting.
-     * @param {Object} setting - The setting which corresponds to the activated
-     * QSS button.
-     * @return {String} The grade name for the `widget` subcomponent.
-     */
-    gpii.psp.qssWidget.getWidgetType = function (setting) {
-        return setting.schema.type === "number" ? "gpii.qssWidget.stepper" : "gpii.qssWidget.menu";
-    };
-
     /**
      * Determines the jQuery element which should be the container of the `widget`
      * view subcomponent depending on the type of the setting.
      * @param {Object} setting - The setting which corresponds to the activated
      * QSS button.
      * @param {jQuery} stepperElement - The container for the QSS stepper widget.
      * @param {jQuery} menuElement - The container for the QSS menu widget.
+     * @param {jQuery} toggleElement - The container for the QSS toggle widget.
      * @return {jQuery} The jQuery element representing the container object.
      */
-    gpii.psp.qssWidget.getWidgetContainer = function (setting, stepperElement, menuElement) {
-        return setting.schema.type === "number" ? stepperElement : menuElement;
+    gpii.psp.qssWidget.getWidgetContainer = function (setting, stepperElement, menuElement, toggleElement) {
+        switch (setting.schema.type) {
```

Comment:
instead of passing each dom element, you could use `that.dom.locate(key)`, where key is the selector for the element

Reply:
Great, this simplifies things a lot and makes everything more generic.